srml/balances/src/lib.rs

```diff
@@ -697,6 +717,7 @@ impl<T: Subtrait<I>, I: Instance> system::Trait for ElevatedTrait<T, I> {
 	type Lookup = T::Lookup;
 	type Header = T::Header;
 	type Event = ();
+	type Error = &'static str;
```

Comment:
Why not `T::Error`?

Reply:
Getting this error with `T::Error` and I can't resolve it. Does the current approach have any issues?
```
Checking srml-balances v2.0.0 (/Users/bryanchen/projects/cennznet/substrate/srml/balances)
error[E0277]: the trait bound `<T as srml_system::Trait>::Error: std::convert::From<Error>` is not satisfied
   --> srml/balances/src/lib.rs:722:35
    |
722 | impl<T: Subtrait<I>, I: Instance> Trait<I> for ElevatedTrait<T, I> {
    |                                   ^^^^^^^^ the trait `std::convert::From<Error>` is not implemented for `<T as srml_system::Trait>::Error`
    |
    = help: consider adding a `where <T as srml_system::Trait>::Error: std::convert::From<Error>` bound
```

Reply:
I expect it'd need to be converted simultaneously with the instance below...